packages/estransform/lib/sourcemaps.js

```diff
@@ -1,8 +1,6 @@
 import { readFile } from 'fs/promises';
 import path from 'path';
-import sourceMapDefault from '@parcel/source-map';
-
-const { default: SourceMapNode } = sourceMapDefault;
+import SourceMapNode from './parcel-source-map-default.cjs';
```

Comment:
I am wondering if we can use the require statement directly in this file, instead of creating .cjs file and build-time banner. I think something like this would work:

```js
import { createRequire } from 'module';

const require = createRequire(import.meta.url);
const SourceMapNode = require('@parcel/source-map').default;
```

Reply:
great suggestion, it does work, solves types issue and I tested it in my project where it works too!
I pushed the update